packages/itwinui-react/src/core/Stepper/WorkflowDiagram.tsx

```diff
@@ -8,23 +8,28 @@ import type { PolymorphicForwardRefComponent } from '../utils/index.js';
 import type { StepperProps } from './Stepper.js';
 import { WorkflowDiagramStep } from './WorkflowDiagramStep.js';
 
-type WorkflowDiagramProps = Pick<StepperProps, 'steps'>;
+type WorkflowDiagramProps = Pick<StepperProps, 'steps'> &
```

Comment:
Instead of picking twice from the same props, you can write it this way

`Pick<StepperProps, 'steps' | 'diagramProps'>`

Reply:
Thanks, I was looking for that :)